docs/hr/content/docs/reusable_snippets/insert_data.ipynb

```diff
@@ -17,21 +17,30 @@
    "outputs": [],
    "source": [
     "# <tab: MongoDB>\n",
-    "from superduperdb import Document\n",
+    "from superduperdb import Document, DataType\n",
     "\n",
-    "def do_insert(data):\n",
-    "    schema = None\n",
-    "    \n",
+    "def do_insert(data, schema = None):\n",
     "    \n",
-    "    if schema is None and (datatype is None  or isinstance(datatype, str)) :\n",
-    "        data = [Document({'x': x}) for x in data]\n",
-    "        db.execute(table_or_collection.insert_many(data))\n",
-    "    elif schema is None and datatype is not None and isintance():\n",
-    "        data = [Document({'x': datatype(x)}) for x in data]\n",
-    "        db.execute(table_or_collection.insert_many(data))\n",
-    "    else:\n",
-    "        data = [Document({'x': x}) for x in data]\n",
-    "        db.execute(table_or_collection.insert_many(data, schema='my_schema'))"
+    "    if isinstance(data[0], dict):  \n",
```

Comment:
This is not how I would do this. I would change the data, so that it has `x` and `y` in it. Then we lose this complicated if statement.